### PR TITLE
Add v3.4.0 docs

### PR DESCRIPTION
**Pull request by xinrong-meng**
Add v3.4.0 docs

### CODE REVIEW

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

File: site/docs/3.4.0/api/java/org/apache/spark/status/protobuf/StoreTypes.ApplicationAttemptInfo.html [hunk 1]
```diff
@@ -0,0 +1,1269 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (gengliangwang) [inline site/docs/3.4.0/api/java/org/apache/spark/status/protobuf/StoreTypes.ApplicationAttemptInfo.html:1]: I think we need to remove these java docs from the protobuf generated files

**Comment by viirya** [comment]
I think @dongjoon-hyun fixed the issue on site/releases/spark-release-3-2-4.html, but CI still failed. Maybe it needs to be rebased?

**Comment by dongjoon-hyun** [comment]
Yes, that's the only failure and I believe we can merge this because we know that issue and this PR only adds a separate directory, @viirya .

**Comment by dongjoon-hyun** [comment]
+1 for rebasing~

File: site/docs/3.4.0/api/java/org/apache/spark/status/protobuf/StoreTypes.ApplicationAttemptInfo.html (hunk 1 above)
Comment (gengliangwang) [reply site/docs/3.4.0/api/java/org/apache/spark/status/protobuf/StoreTypes.ApplicationAttemptInfo.html:1]: We can merge this giant PR first and then remove these files.

**Review by gengliangwang — APPROVED** [review]

**Review by viirya — APPROVED** [review]

**Comment by gengliangwang** [comment]
Thanks for the work. I am merging this one.

**Comment by xinrong-meng** [comment]
Thank you all!

**Comment by dongjoon-hyun** [comment]
Also, cc @sunchao .

**Review by sunchao — COMMENTED** [review]
late LGTM

**Comment by yaooqinn** [comment]
late LGTM

**Pull request merged**